packages/mask/src/resources/NETSWAPIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import { SvgIconProps, SvgIcon } from '@mui/material'
+
+const svg = (
+    <svg version="1.1" viewBox="0 0 26 26" width="26px" height="26px" xmlns="http://www.w3.org/2000/svg">
```

Comment:
plz use https://aka.pw/svgomg reduce the svg image

Reply:
ok!